index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Comforter+Brush&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+<h1 id="title" class="text-center">Survey Form</h1>
+<p id="description">Thank you for taking the time to fill out this form </p>
+
+
+<form action="#" id="survey-form">
+
+
+<label for="name" id="name-label">Enter your name:</label>
+<input type="text" id="name" placeholder="Enter a Name" required> <br> <br>
+
+
+
+<label for="email" id="email-label">Enter your email address:</label>
+<input type="email" id="email" placeholder="Enter an e-mail" required > <br><br>
+
+<label for="phone-number" id="number-label">Enter your phone-number:</label>
+<input type="number" id="number" placeholder="+1 222 222 2222" min="10" max="99" required><br> <br>
+
+
+
+<select name="pets" id="dropdown" required>
+  <option value="spider">option 1</option>
+  <option value="goldfish">option 2</option>
+</select> <br><br>
+
+
+<label>
+<input id="name" type="radio" name="gender" value="Male" class="gn">
+Male 
+</label>
+
+<label>
+<input id="name" type="radio" name="gender" value="Female" class="gn"> Female
+
+</label>
+<br>
+
+ 
+  <input type="checkbox" id="option1" name="option1.1" value="1">
+  <label for="vehicle1">Checkbox 1</label><br>
+ 
+  <input type="checkbox" id="option2" name="options1.2" value="2">
+  <label for="vehicle2">Checkbox 2</label><br>
+
+  <input type="checkbox" id="option3" name="option1.3" value="3">
+  <label for="vehicle3">Checkbox 3</label><br><br>
+
+   
+  <br><br>
+
+  <textarea id="t" name="t" placeholder="Leave us a comment!" rows="5" cols="40"></textarea>
+ 
+ <br><br>
+
+
+<input type="submit" id="submit" value="Push Me!">
+
+</form>
+
+
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js">
+
+    </script>
```

Comment:
This repository is not supposed to have Scripts in it. 
Please remove, and stick to the instructions

***It was required, it is what checks that all requirements were met.***